Jenkinsfile

```diff
@@ -42,41 +42,42 @@
 // Hashtag in the source to build current CI docker builds
 //
 //
+import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
 // NOTE: these lines are scanned by docker/dev_common.sh. Please update the regex as needed. -->
-ci_lint = "tlcpack/ci-lint:v0.67"
-ci_gpu = "tlcpack/ci-gpu:v0.77"
-ci_cpu = "tlcpack/ci-cpu:v0.77"
-ci_wasm = "tlcpack/ci-wasm:v0.71"
-ci_i386 = "tlcpack/ci-i386:v0.73"
-ci_qemu = "tlcpack/ci-qemu:v0.08"
-ci_arm = "tlcpack/ci-arm:v0.06"
+ci_lint = 'tlcpack/ci-lint:v0.67'
```

Comment:
There are lots of format change noises in this file and they seems not relate to this PR. Is it required or just done by auto-formatting? I would suggest removing them if not required.

Reply:
+1 would be great to avoid too many formatting changes. in particular this one also requires updates to docker/dev_common.sh

Reply:
I passed the Jenkisfile through a Groovy formatter, but +1 I reverted the changes to keep the diff down.